common/src/main/java/com/hedera/block/common/utils/FileUtilities.java

```diff
@@ -62,21 +65,22 @@ public final class FileUtilities {
                             PosixFilePermission.OTHERS_EXECUTE));
 
     /**
-     * Log message template used when a path is not created because a file
-     * or folder already exists at the requested path.
+     * Log message template used when a path is not created because a file or folder already exists
+     * at the requested path.
      */
     private static final String PRE_EXISTING_FOLDER_MESSAGE =
             "Requested %s [%s] not created because %s already exists at %s";
 
     /**
-     * Create a new path (folder or file) if it does not exist.
-     * Any folders or files created will use default permissions.
+     * Create a new path (folder or file) if it does not exist. Any folders or files created will
+     * use default permissions.
      *
-     * @param toCreate valid, non-null instance of {@link Path} to be created
-     * @param logLevel valid, non-null instance of {@link System.Logger.Level} to use
+     * @param toCreate         valid, non-null instance of {@link Path} to be created
+     * @param logLevel         valid, non-null instance of {@link System.Logger.Level} to use
      * @param semanticPathName valid, non-blank {@link String} used for logging that represents the
-     *     desired path semantically
-     * @param createDir {@link Boolean} value if we should create a directory or a file
+     *                         desired path semantically
+     * @param createDir        {@link Boolean} value if we should create a directory or a file
+     *
```

Comment:
Same here, do not try to align descriptions.
For Javadoc, we should use one space after the parameter _name_, and 4 space indentation(only) on subsequent lines.

note, if your IDE is doing this, change it to not reformat javadoc this way (IDE formatting doesn't match spotless anyway, so it is just always going to be wrong).

Reply:
✅ I reversed the formatting on the docs everywhere. It was indeed a problem with the IDE, I do use only spotlessApply, but looks like the IDE formatting also went through and I thought that was spotless that did that, sorry about that, I should not have this problem in the future.

Reply:
IDE defaults are a pain sometimes...